_SQLServer/TableLoadHeroes.sql

```diff
@@ -0,0 +1,478 @@
+'http://htkb.info/SDE/1.0/Cards/Heroes/Card_Hero_Hearthsworn_Fighter_1.0_back.png', 
+'http://htkb.online/SDWiki/SDE/1.0/Cards/Heroes/Hearthsworn_Fighter.xhtml', 
+'Hero', '1.0', '1.0 Core Set', 'Default',   
+'Resilient and unflinching in the face of combat, the Dwarven Hearthsworn are warriors without peer.  Able to withstand injury that would fell any of the lesser races the Hearthsworn cleave through their adversaries bellowing insults and challenges to any foolish enough to face them.');
+insert into Characters (CardIndex) VALUES (0);
+insert into KeywordAssignments (CharacterIndex, KeywordIndex) VALUES (1, 0); --! To look at later
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 0); --Dwarven Curse
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 0); --Cleave
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 1); --Hero's Balm
+insert into ExploreCharacters (CharacterIndex, ModelSize, CreatureType, Movement, Actions, Strength, Armor, Will, Dexterity, Health, Potions) VALUES (0, 'Small', 'Dwarf Hero', 6, 3, '2B1R', '2R', '2B', '2B', 6, 1);
+insert into OffenseAssignments (CharacterIndex, AttributeIndex, OffenseRange) VALUES (0,0,1);--! To look at later
+insert into DefenseAssignments (CharacterIndex, AttributeIndex) VALUES (0,2);--! To look at later
+
+
+--Hexcast Sorceress
+insert into Cards (Name, PictureFront, PictureBack, Link, CardType, ProductSet, Module, Mode, Flavor) VALUES ('Hexcast Sorceress', 
+'http://htkb.info/SDE/1.0/Cards/Heroes/Card_Hero_Hexcast_Sorceress_1.0.png', 
+'http://htkb.info/SDE/1.0/Cards/Heroes/Card_Hero_Hexcast_Sorceress_1.0_back.png', 
+'http://htkb.online/SDWiki/SDE/1.0/Cards/Heroes/Hexcast_Sorceress.xhtml', 
+'Hero', '1.0', '1.0 Core Set', 'Default',   
+'Trained in the dark art of hexes and curses the Hexcast Covens are uneasy allies to the rulers of Crystalia.  Nonetheless their potency in battle cannot be denied.  Slowing enemies, shattering armor, and weakening their foes a Hexcast Sorceress can lay even the mightest low.');
+insert into Characters (CardIndex) VALUES (0);
+insert into KeywordAssignments (CharacterIndex, KeywordIndex) VALUES (1, 0); --! To look at later
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 0); --Frozen Feel
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 0); --Hobbled Hands
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 1); --Broken Bones
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 1); --Wizard Wings
+insert into ExploreCharacters (CharacterIndex, ModelSize, CreatureType, Movement, Actions, Strength, Armor, Will, Dexterity, Health, Potions) VALUES (0, 'Small', 'Human Hero', 6, 3, '2B', '3B', '1B1R', '1B1R', 5, 1);
+insert into OffenseAssignments (CharacterIndex, AttributeIndex, OffenseRange) VALUES (0,3,1);--! To look at later
```

Comment:
OffenseRange should be 8

This character should have a 1st offense assignment for attack (0,0,1)